### PR TITLE
Update index.jsx

### PR DESCRIPTION
**Pull request by senchenko-oleksiy**
- **Add possibility to set custom className to input element**

### CODE REVIEW

**Comment by senchenko-oleksiy** [comment]
> Can you please elaborate on what change exactly did you make here?

add possibility to assign custom className to input, not only inline styles

File: src/lib/index.jsx [hunk 1]
```diff
@@ -89,6 +89,7 @@ class SingleOtpInput extends PureComponent<*> {
             hasErrored && isStyleObject(errorStyle) && errorStyle
           )}
           className={this.getClasses(
+            inputStyle && inputStyle,
```
Comment (ajayns) [inline src/lib/index.jsx]: This does not require a check: `inputStyle && inputStyle` -> `inputStyle`

**Comment by ajayns** [comment]
Sorry about the delay, please do make the changes, show that the changes are working fine with the codesandbox mentioned in the readme and we'll get this merged :) @senchenko-oleksiy

**Comment by akashnimare** [comment]
@senchenko-oleksiy would you be interested in finishing this PR?

**Comment by apollonian** [comment]
Closing the PR due to inactivity.
Please open a new pull request if you'd still like to work on it :)

**Pull request closed without merging**